lib/timescaledb/rails/extensions/active_record/schema_statements.rb

```diff
@@ -1,12 +1,27 @@
 # frozen_string_literal: true
 
-require 'active_record/connection_adapters/postgresql_adapter'
-
 module Timescaledb
   module Rails
     module ActiveRecord
       # :nodoc:
       module SchemaStatements
+        # Returns an array of hypertable names defined in the database.
+        def hypertables
+          query_values('SELECT hypertable_name FROM timescaledb_information.hypertables')
+        end
+
+        # Checks to see if the hypertable exists on the database.
+        #
+        #   hypertable_exists?(:developers)
+        #
+        def hypertable_exists?(hypertable)
```

Comment:
you could also consider to rename this to just `hypertable?`

Reply:
I tried to follow rails naming convention here, check here -> https://github.com/rails/rails/blob/b8d9a363201717e493c3d765d1240f2e3052e4a3/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L59